ui/src/components/inputs/EditorSidebar.vue

```diff
@@ -517,8 +567,20 @@
 
                     this.selectedFiles = flatList.slice(start, end + 1).map(item => item.path);
                     this.selectedNodes = flatList.slice(start, end + 1).map(item => item.id);
+
+                } else if (window.event.ctrlKey || window.event.metaKey) {
```

Comment:
This will return true on windows, if the windows key is entered. 
Is this intened?

If not, you can always ask for the userAgents OS and deal with it this way.